update_version_templates.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Can this be put in the contrib folder please?

Let's call it replace_in_files.sh instead?